### PR TITLE
Vultraz admin ui nav tweak

### PR DESCRIPTION
**Pull request by MadeByMike**
Some change to implementation of #1868.

Also fixes the use of some outdated\bad practice flex properties.

> Gets rid of the margin around the bottom nav buttons so they're easier to click. Also brings them in line with the list items above that extend to the screen edge.

### CODE REVIEW

File: packages/arch/packages/navbar/src/PrimaryNav.js [hunk 1]
```diff
@@ -108,6 +107,27 @@ export const PrimaryNavItem = styled(ItemElement)(({ depth, isSelected, mouseIsO
...
+  return {
+    color: mouseIsOverNav ? colors.N70 : colors.N40,
+    padding: PRIMARY_NAV_GUTTER,
+    textDecoration: 'none',
+    flexGrow: 1,
+    flexBasis: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    ':hover': {
```
Comment (jordanoverbye) [inline packages/arch/packages/navbar/src/PrimaryNav.js]: I think this key should be `':hover, :focus'` for an easy a11y win

File: packages/arch/packages/navbar/src/PrimaryNav.js [hunk 2]
```diff
@@ -75,16 +73,17 @@ export const PrimaryNavItem = styled(ItemElement)(({ depth, isSelected, mouseIsO
...
     position: 'relative',
     textDecoration: 'none',
     textOverflow: 'ellipsis',
     whiteSpace: 'nowrap',
     transition: 'color 110ms',
 
+    flexGrow: 1,
+    flexBasis: '100%',
+
     ':hover': {
```
Comment (jordanoverbye) [inline packages/arch/packages/navbar/src/PrimaryNav.js]: I think this key should be `':hover, :focus'` for an easy a11y win

Comment (MadeByMike) [inline packages/arch/packages/navbar/src/PrimaryNav.js]:
```suggestion
    ':hover, :focus': {
```

File: packages/arch/packages/navbar/src/PrimaryNav.js (hunk 1 above)
Comment (MadeByMike) [inline packages/arch/packages/navbar/src/PrimaryNav.js]:
```suggestion
    ':hover, :focus': {
```

**Pull request merged**